### PR TITLE
Read me

### PR DESCRIPTION
**Pull request by AshutoshM29**
Raised a PR for the Read Me file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,226 @@
...
+    - Success Button
+    - Error Button
+
+- Outline Buttons
+
+We have 4 types of Solid Buttons
+    - Primary Button
+    - Secondary Button
+    - Success Button
+    - Error Button
```
Comment (Sahil25061999) [inline README.md]: This is typo i guess

**Review by JuvitaSaini09 — APPROVED** [review]
looks good to merge

File: README.md [hunk 2]
```diff
@@ -1,2 +1,226 @@
...
+
+9. [Lists](#lists)
+
+10. [Navigation](#navigation)
+
+11. [Modal](#modal)
+
+12. [Toast](#toast)
+
+13. [Rating](#rating)
```
Comment (Sahil25061999) [inline README.md]: I would suggest to give links to the component in the documentation page, but its upto you, not a necessary change.

**Review by Sahil25061999 — APPROVED** [review]
I think you should also provide a link for the original documentation site.

**Pull request merged**